core/tests/performance_tests/test_config.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Performance test config."""
```

Comment:
Please be specific on what 'configuration' is. If I'm a developer that wants to implement a new performance test, I might look at existing tests for reference. I'll notice this file is used, but I will have no idea what sort of data should go here. I might just make it up as I go along and, if we're lucky, I'll be right. In the more likely circumstance, I may introduce constants here that you aren't expecting and that will result in this file becoming ambiguous and confusing.

Having more specific documentation is very helpful for maintenance. Consider explaining that perhaps this file should only contain configuration values related to performance metrics, such as DOM initialization times, total request transfer sizes for a single session, etc. It doesn't have to be exhaustive, but the documentation should paint a clear picture such that a developer may infer what additional values to put here.